src/invoices/InvoiceDetails/Information/Information.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import Information from './Information';
+
+jest.mock('../../../common/components/Status/StatusValue', () => {
+  return () => <span>StatusValue</span>;
+});
+
+jest.mock('../BatchGroupValue', () => {
+  return () => <span>BatchGroupValue</span>;
+});
+
+const invoiceInformation = {
+  batchGroupId: 'cd592659-77aa-4eb3-ac34-c9a4657bb20f',
+  invoiceDate: '2020-12-09T00:00:00.000+0000',
+  status: 'Open',
+  source: 'User',
+  currency: 'USD',
+};
+
+const renderInformation = ({
+  batchGroupId,
+  invoiceDate,
+  status,
+  source,
+  currency,
+  lockTotal,
+}) => (render(
+  <Information
+    batchGroupId={batchGroupId}
+    invoiceDate={invoiceDate}
+    status={status}
+    source={source}
+    currency={currency}
+    lockTotal={lockTotal}
+  />,
+));
+
+describe('Information component', () => {
+  it('should display invoice information labels', () => {
+    const { getByText } = renderInformation(invoiceInformation);
+
+    expect(getByText('ui-invoice.invoice.details.information.invoiceDate')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.paymentDue')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.paymentTerms')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.approvedDate')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.source')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.note')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.totalUnits')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.subTotal')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.adjustment')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.calculatedTotalAmount')).toBeDefined();
+    expect(getByText('ui-invoice.invoice.details.information.subTotal')).toBeDefined();
```

Comment:
I'm not sure this makes sense to check every label. One is enough to be sure component is rendered. However, even this might not make sense, since it's better to check that value of some field is rendered.